doc/source/reshaping.rst

```diff
@@ -698,10 +703,124 @@ handling of NaN:
     In [3]: np.unique(x, return_inverse=True)[::-1]
     Out[3]: (array([3, 3, 0, 4, 1, 2]), array([nan, 3.14, inf, 'A', 'B'], dtype=object))
 
-
 .. note::
     If you just want to handle one column as a categorical variable (like R's factor),
     you can use  ``df["cat_col"] = pd.Categorical(df["col"])`` or
     ``df["cat_col"] = df["col"].astype("category")``. For full docs on :class:`~pandas.Categorical`,
     see the :ref:`Categorical introduction <categorical>` and the
     :ref:`API documentation <api.categorical>`.
+
+Frequently Asked Questions (and Examples)
+------------------
+
+In this section, we will review frequently asked questions and examples. The 
+column names and relevant column values are named to correspond with how this
+DataFrame will be pivoted in the answers below.
+
+.. ipython:: python
+
+   np.random.seed([3,1415])
+   n = 20
+   
+   cols = np.array(['key', 'row', 'item', 'col'])
+   arr1 = (np.random.randint(5, size=(n, 4)) // [2, 1, 2, 1]).astype(str)
+   
+   df = pd.DataFrame(np.core.defchararray.add(cols, arr1), columns=cols).join(
+       pd.DataFrame(np.random.rand(n, 2).round(2)).add_prefix('val')
+   )
+
+   df
+
+Question 1
+~~~~~~~~~~
+
+How do I pivot ``df`` such that the ``col`` values are columns,
```

Comment:
Double backticks are for literals, where single backticks are for inline code / argument refs. Can you change any argument reference (i.e. `df`, `col`, etc...) to single backticks?

Reply:
@WillAyd while that makes sense, this seems inconsistent with how the single backticks and double backticks are being used elsewhere in this doc. It also seems like the double backticks look better in the docs itself.